packages/universal-wallet-fastify-plugin/src/walletRoutes.ts

```diff
@@ -4,11 +4,14 @@ import issuer from './routes/issuer';
 import holder from './routes/holder';
 import exchange from './routes/exchange';
 import verifier from './routes/verifier';
+import base from './routes/base';
 import { APIEnum, DiscoveryEnum, WalletOptions } from './types/WalletOptions';
 
 export const walletRoutes = (options: WalletOptions) => (
   fastify: FastifyInstance
 ) => {
+  fastify.register(base(options));
```

Comment:
Would rather we:
1) add optionality for the base endpoints
2) make it abundantly clear in the documentation, and in the code with a warning that a preValidation hook implementing authorization should be provided, so that `.add` and `.get` methods aren't exposed in the wild

Reply:
lets make it configurable... I am feeling we may actually want to relabel the configuration to allign more with the spec interface names.

Reply:
we made it configurable